overlord/managers_test.go

```diff
@@ -1034,7 +1034,7 @@ func (s *mgrsSuite) TestHappyRemoteInstallAndUpdateWithEpochBump(c *C) {
 	chg.AddAll(ts)
 
 	st.Unlock()
-	err = s.o.Settle(settleTimeout)
+	err = s.o.Settle(2 * settleTimeout)
```

Comment:
Should we simply bump settleTimeout, affecting all other managers tests, or is it just these 2 tests that are very slow?

Reply:
So far it looks like this one (TestHappyRemoteInstallAndUpdateWithEpochBump) in particular is slow so I took the easy path and just bumped it for this one. I have no strong opinion about raining it in general, that's fine with me.

Reply:
I actually just saw this for `TestHappyRemoteInstallAndUpdateManyWithEpochBumpAndOneFailing` too so I'd say go for @stolowski's suggestion and bump it up everywhere if it's not an issue

Reply:
Thanks, good point. I increased the global settleTimeout now.